hypervisor/dm/vpci/vhostbridge.c

```diff
@@ -48,43 +48,26 @@ static void init_vhostbridge(struct pci_vdev *vdev)
 	/* PCI config space */
 	pci_vdev_write_vcfg(vdev, PCIR_VENDOR, 2U, 0x8086U);
 	pci_vdev_write_vcfg(vdev, PCIR_DEVICE, 2U, 0x5af0U);
-
```

Comment:
refine the init_vhostbridge to use DWORD operations to simplify the initialization operation. LOC is an additional gain.